README-dev.md

```diff
@@ -131,100 +168,88 @@ let g:syntastic_ocaml_checkers=['merlin']
-$ dune exec src/app/cli/src/mina.exe -- libp2p generate-keypair --privkey-path /path/to/key
-```
+1. Generate a key pair so that the daemon can create an account to issue blocks from using the same `mina.exe` binary:
+
+  ```shell
+  $ dune exec src/app/cli/src/mina.exe -- libp2p generate-keypair --privkey-path /path/to/key
+  ```
+
+When prompted, enter a passphrase. During development, you can leave it blank for convenience, but using a passphrase is strongly encouraged when running a real node! 
 
-The program will ask you for a passphrase, which can be left blank for convenience (of course this is
-highly discouraged when running a real node!). The running daemon expects to find this passphrase in
-an environment variable `MINA_LIBP2P_PASS`, which needs to be defined even if the passphrase is empty.
-Furthermore, `/path/to/key` needs to belong to the user running the daemon and to have filesystem
-permissions set to `0600`. Also the directory containing the file (`/path/to`) needs its permissions
-set to `0700`.
+The running daemon expects to find this passphrase in
+an environment variable `MINA_LIBP2P_PASS`, which must be defined even if the passphrase is empty.
+The `/path/to/key` must belong to the user running the daemon. Set these filesystem permissions:
 
 ```shell
 $ chmod 0600 /path/to/key
 $ chmod 0700 /path/to
 ```
 
-Additionally, it's necessary to provide a list of peers to connect to to bootstrap the node.
-The list of peers depends on the network one wants to connect to and is announces when the
-network is being launched. For mainnet the list can be found at:
-https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt .
+Additionally, you must provide a list of peers to connect to to bootstrap the node.
+The list of peers depends on the network you want to connect to and is announced when the network is being launched. For Mainnet, the list of peers is avaialable at:
+https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt.
 
-Moreover, `daemon.json` config file also contains some bootstrap data specific to the network
-the node is trying to connect to. Therefore it also needs to be tailored specifically for
-a particular network. This file can also override some of the configuration options selected
-during compilation (see above). At the moment it can be extracted from the docker image
-dedicated to running a particular network. If it's not located in the config directory,
-it can be pointed to with `--config-file` option.
+The `daemon.json` config file also contains bootstrap data that is specific to the network the node is trying to connect to and must be tailored specifically for
+a particular network. This file can also override some of the configuration options selected during compilation. The `daemon.json` file can be extracted from the Docker image
+that is dedicated to running a particular network. If it's not located in the `config` directory, it can be pointed to with `--config-file` option.
```

Comment:
```suggestion
that is dedicated to running a particular network. If it's not located in the `config` directory, it can be pointed to with `--config-file` option.
```

Reply:
@nicc  line breaks were already present in the files, tbh I don't know the impact or value